assets/all.settings.php

```diff
@@ -13,6 +13,20 @@
 
 use Drupal\Core\Installer\InstallerKernel;
 
+// Disable the APCu class cache. This is needed in order to move the
+// asset_injector module from local to contrib modules. The APCu class cache
+// keeps track of which classes exists and where, and need to be flushed when
+// we move the module to avoid fatal errors. But the APCu cache exists within
+// the PHP process, so there's no way to clear the cache in the PHP-FPM
+// processes serving the sites from the Drush command doing the update. So we
+// disable the cache for as long as the local version exists. As the cache is
+// obviously cleared at deploy time, this keeps the cache out of the game
+// until the deploy hook has had time to get rid of it.
+// See dpl_webmaster.install/deploy.php.
+if (file_exists('modules/local/asset_injector')) {
```

Comment:
so eventually, these lines of code wont have any affect? cause you'll delete `modules/local/asset_injector`

I understand your comment above, but it does worry me a bit to touch something so central - but i trust you know what you're doing.